spec/models/profile_spec.rb

```diff
@@ -0,0 +1,32 @@
+RSpec.describe Profile do
+  describe "validations" do
+    subject(:profile) { build(:profile) }
+
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:cell_phone) }
+    it { is_expected.to validate_presence_of(:birthday) }
+    it { is_expected.to validate_numericality_of(:weight).is_greater_than(0) }
+    it { is_expected.to validate_numericality_of(:height).is_greater_than(0) }
+    it { is_expected.to belong_to :user }
+
+    context "when birthday" do
+      it "is valid with past birthday" do
+        profile.birthday = Time.zone.today - 1
+        profile.valid?
+        expect(profile.errors.keys).not_to include :birthday
+      end
+
+      it "is valid with current birthday" do
+        profile.birthday = Time.zone.today
+        profile.valid?
+        expect(profile.errors.keys).not_to include :birthday
+      end
+
+      it "is invalid with future birthday" do
+        profile.birthday = Time.zone.today + 1
+        profile.valid?
+        expect(profile.errors.keys).to include :birthday
+      end
+    end
```

Comment:
Acho que não entendeu o que eu quis dizer no comentário anterior, mas saca só como da pra fazer:

```suggestion
describe "birthday" do
  before do
		profile.birthday = date
		profile.valid?
  end
  
	context "when is in the past" do
    let(:date) { 1.day.ago }
    
		it "does not add an error" do
			expect(profile.errors.keys).not_to include :birthday
		end
	end

	context "when is today" do
		let(:date) { Time.zone.today }
    
		it "does not add an error" do
			expect(profile.errors.keys).not_to include :birthday
		end
	end

	context "when is in the future" do
    let(:date) { 1.day.from_now }
    
		it "adds an error" do
			expect(profile.errors.keys).to include :birthday
		end
	end
end
```

Separei melhor pra tu ver, mas acho que ele não vai permitir fazer dentro de validations por ter contexts aninhados demais, por isso, o jeito vai ser separar de novo igual tinha feito colocando o describe "birthday"

Reply:
Vai ter que ajustar a identação, fiz no editor e colei aqui, ficou zoado, se não der pra entender me avisa

Reply:
![image](https://user-images.githubusercontent.com/37879125/101781573-c0fab400-3ad6-11eb-9778-7a9e24ea423f.png)

Reply:
Refatorei conforme você informou !